src/data/ACTIONS/SMN.js

```diff
@@ -31,13 +42,15 @@ export default {
 		name: 'Energy Siphon',
 		icon: 'https://xivapi.com/i/002000/002697.png',
 		cooldown: 30,
+		cooldownGroup: SMN_COOLDOWN_GROUP.ENERGY,
 	},
 
 	ENKINDLE: {
 		id: 184,
 		name: 'Enkindle',
 		icon: 'https://xivapi.com/i/002000/002677.png',
 		cooldown: 120,
+		cooldownGroup: SMN_COOLDOWN_GROUP.ENKINDLE, //this skill isn't techically usable anymore, but would be part of this group
```

Comment:
If the skill isn't usable, probably better off removing the data - there isn't a reason to keep it sitting around.